packages/core/src/provider/shared.ts

```diff
@@ -174,6 +175,10 @@ function formatErrorMessage(errors: string[]) {
     color.red(isTerserError ? 'Minify error: ' : 'Compile error: '),
   );
 
+  if (!errors.length) {
+    return `${title}\n${color.yellow(`For more details, please setting 'stats.errors: true' `)}`;
```

Comment:
Why do users need to manually set `stats.errors: true`

Reply:
not need, unless user manually  set `stats.errors: false` or `stats.preset: none`.